modules/efk-logging-deploy-subscription.adoc

```diff
@@ -82,12 +82,14 @@ metadata:
   name: openshift-logging
   annotations:
     openshift.io/node-selector: "" <1>
-  labels:
+  labels:  <2>
     openshift.io/cluster-logging: "true"
     openshift.io/cluster-monitoring: "true"
 ----
-<1> Optionally specify an empty node selector in order for the logging pods to spread
-evenly across your cluster. If you want the logging pods to run on specific nodes, you can specify a node selector value here.
+<1> Specify an empty node selector in order for the logging pods to spread evenly across your cluster. 
+If you want the logging pods to run on specific nodes, leave this parameter empty 
+and specify node selectors in the Cluster Logging Custom Resource after installation.
```

Comment:
@anpingli Can you comment on this change? The user should use _openshift.io/node-selector: ""_ in the cluster logging project. If the user wants to use node selectors,  modify the CR.

Reply:
@xltian 
Can you comment on this change? The user should use openshift.io/node-selector: "" in the cluster logging project. If the user wants to use node selectors, modify the CR.